operators/Makefile

```diff
@@ -237,22 +237,34 @@ ifndef GCLOUD_PROJECT
 	$(error GCLOUD_PROJECT not set)
 endif
 
-credentials: require-gcloud-project
-	GKE_CLUSTER_VERSION=$(GKE_CLUSTER_VERSION) ./hack/gke-cluster.sh credentials
+DEPLOYER=./hack/deployer/deployer --plans-file=hack/deployer/config/plans.yml --run-config-file=hack/deployer/config/run-config.yml
```

Comment:
I'm wondering whether we should maybe move everything related to GKE/Minikube/Deployer away from the main Makefile (into eg. `hack/deployer/Makefile`) for better separation of concerns?

Reply:
That's something that we can look into, but most of the deployer related code is just single line deployer invocations. I'd be careful to separate if it would only make us switch from calling $(DEPLOYER) to calling $(MAKE). Unless you mean to remove those targets from this file altogether, but then I'd have concerns about usability (cding back and forth). WDYT?

Reply:
Yeah I meant removing all targets. Let's keep it in a single Makefile for now, we'll see later if that becomes a problem :)